Frontend/lib/screens/auth/login_screen.dart

```diff
@@ -24,14 +25,26 @@ class _LoginScreenState extends State<LoginScreen> {
       Get.put(AuthenticationController());
   final _formKey = GlobalKey<FormState>();
 
-  SignInVariable() {
+  signInVariable() {
     return {
       'username': emailController.text,
       'password': passwordController.text,
     };
   }
 
   bool _obscureText = true;
+
+  @override
+  void initState() {
+    super.initState();
+
+    //Todo: Remove this
+    if (kDebugMode) {
```

Comment:
Please can you explain this? if it is not required, can you remove it then?

Reply:
The app is still on debug mode, it make work faster and I will remove it during clean up.

Reply:
okay, I am asking this because we have been using this same email and password to login, do you mind if we jump on a call in the morning to try something else?

Reply:
Sure, by 9am

Reply:
Alright,till then